app/scripts/lib/SnapsNameProvider.ts

```diff
@@ -12,6 +12,7 @@ import {
   AddressLookupResult,
   Snap as TruncatedSnap,
 } from '@metamask/snaps-sdk';
+// @ts-expect-error see: https://github.com/MetaMask/snaps/pull/2174
```

Comment:
snaps team told me they don't want to rush this change because they don't want to break things, so we're stuck with this comment for now.